routes/web.php

```diff
@@ -17,20 +17,22 @@
 |
  */
 
+Route::middleware('auth')->group(function () {
+    Route::get('profile/index', [UserController::class, 'index'])->name('profile');
+    Route::post('profile/update', [UserController::class, 'update'])->name('update.profile');
+    Route::post('social_network/update', [SocialNetworkController::class, 'update'])->name('update.social_network');
+    Route::get('wish-list/index', [WishListController::class, 'index'])->name('my_wish_list');
+    Route::post('wish-list/create', [WishListController::class, 'create'])->name('create.wishList');
```

Comment:
very good thanks for the effort.

you could make the route name as hierarchy but its fine right now.